<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeDetectionDialogBase</class>
 <widget class="QDialog" name="ChangeDetectionDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Detection Plugin</string>
  </property>
  <widget class="QLabel" name="map_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>600</width>
     <height>552</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_coords">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>340</y>
     <width>181</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск изменений</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>70</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="results_viewer_button">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>400</y>
     <width>181</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Просмотр данных</string>
   </property>
  </widget>
  <widget class="QComboBox" name="start_year">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>280</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2019</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2020</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2022</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="completion_year">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>280</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2022</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2020</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="download_button">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>460</y>
     <width>181</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить продукты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>250</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Начальный год</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>250</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Конечный год</string>
   </property>
  </widget>
  <widget class="QPushButton" name="train_net_button">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>520</y>
     <width>181</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Тренировка нейронной сети</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cloud_cover">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>210</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="value">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Облачность</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
